libs/designer/src/lib/ui/panel/recommendation/recommendationPanelContext.tsx

```diff
@@ -49,23 +49,39 @@ export const RecommendationPanelContext = (props: RecommendationPanelContextProp
 
   const [selectionState, setSelectionState] = useState<SelectionState>(SELECTION_STATES.SEARCH);
 
-  const selectedOperationId = useSelectedSearchOperationId();
   const { data: allOperations, isLoading: isLoadingOperations } = useAllOperations();
-  const selectedOperation = allOperations.find((o) => o.id === selectedOperationId);
+  const [selectedOperation, setSelectedOperation] = useState<DiscoveryOperation<DiscoveryResultTypes> | undefined>(undefined);
+  const [isLoadingOperationsGroup, setIsLoadingOperationGroup] = useState<boolean>(false);
```

Comment:
Sorry but one last nit, the set is fine but the reference still has an S
```suggestion
  const [isLoadingOperationGroup, setIsLoadingOperationGroup] = useState<boolean>(false);
```

Reply:
Thanks for the catch. Done.